content/millennium/r4/individuals/related-person.md

```diff
@@ -111,8 +111,124 @@ List an individual RelatedPerson by its id:
 
 ### Errors
 
+## Create
+
+Create a new Encounter Level RelatedPerson.
+
+    POST /RelatedPerson
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
```

Comment:
I believe we do fail on unsupported fields now.

Reply:
updated here:https://github.com/cerner/fhir.cerner.com/pull/527/commits/91acf8cf0ff17eb8973b5b12811428fec2b129c6#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR122